File: customers-staff-master/src/Front/src/api/new/dto/WorkOrder/WorkOrderDto.ts
import { DateTime } from "../DateTime";
import { Guid } from "../Guid";
import { CustomerProductDto } from "./CustomerProductDto";
import { CustomerValueListDto } from "./CustomerValueListDto";
import { ShopProductDto } from "./ShopProductDto";
import { ShopRequisitesDto } from "./ShopRequisitesDto";
import { ShopServiceDto } from "./ShopServiceDto";
import { WorkOrderNumberDto } from "./WorkOrderNumberDto";
import { WorkOrderStatusDto } from "./WorkOrderStatusDto";

export interface WorkOrderDto {
  number: WorkOrderNumberDto;
  shopRequisites: ShopRequisitesDto;
  receptionDate: DateTime;
  completionDatePlanned: DateTime;
  completionDateFact?: DateTime;
  warrantyNumber?: string;
  receptionWorkerId?: Guid;
  status: WorkOrderStatusDto;
  clientId: Guid;
  additionalText?: string;
  customerValues: CustomerValueListDto;
  customerProducts: CustomerProductDto[];
  shopProducts: ShopProductDto[];
  shopServices: ShopServiceDto[];
}

File: customers-staff-master/src/CustomersAndStaff/FrontApi.Dto/WorkOrders/WorkOrderDto.cs
using System;

namespace Market.CustomersAndStaff.FrontApi.Dto.WorkOrders
{
    public class WorkOrderDto
    {
        public WorkOrderNumberDto Number { get; set; }
        public ShopRequisitesDto ShopRequisites { get; set; }
        public DateTime ReceptionDate { get; set; }
        public DateTime CompletionDatePlanned { get; set; }
        public DateTime? CompletionDateFact { get; set; }
        public string WarrantyNumber { get; set; }
        public Guid? ReceptionWorkerId { get; set; }
        public WorkOrderStatusDto Status { get; set; }
        public Guid ClientId { get; set; }
        public CustomerValueListDto CustomerValues { get; set; }
        public CustomerProductDto[] CustomerProducts { get; set; }
        public ShopProductDto[] ShopProducts { get; set; }
        public ShopServiceDto[] ShopServices { get; set; }
        public DateTime WarrantyDate { get; set; }
        public string AdditionalText { get; set; }
    }
}